README.md

```diff
@@ -96,7 +96,7 @@ To get started with contributing:
 
 ## Community
 
-* [Discord](https://discord.gg/rooch)
+* [Discord](https://discord.gg/hpJZkBFggN)
```

Comment:
Is this link permanently valid?

Reply:
I add more boost to the discord server, now the https://discord.gg/rooch link is working.